src/popup/components/Card.vue

```diff
@@ -114,6 +115,15 @@ export default {
   background-size: cover;
   transition: $transition-interactive;
 
+  &-darken::before {
+    content: '';
+    position: absolute;
+    z-index: -1;
+    inset: 0;
+    background-color: var(--screen-bg-color);
+    opacity: 0.7;
+  }
```

Comment:
Is this still relevant with the new background image for the DAPPS browser?
Resolve if yes.